docs/spec/spec.md

```diff
@@ -176,3 +256,16 @@ a Channel system that receives and delivers events._
 | channel\* | ObjectRef | The continuation Channel for the link. | Must be a Channel. |
 
 \*: Required
+
+### TriggerFilter
+
+| Field         | Type                      | Description                                        | Constraints |
+| ------------- | ------------------------- | -------------------------------------------------- | ----------- |
+| sourceAndType | TriggerFilterSourceAndTpe | A filter that can specific both a source and type. |             |
+
+### TriggerFilterSourceAndTpe
+
+| Field  | Type   | Description                             | Constraints                          |
+| ------ | ------ | --------------------------------------- | ------------------------------------ |
+| source | String | Event source as defined by CloudEvents. | Also allowed to be the string 'Any'. |
+| type   | String | Event type as defined by CloudEvents.   | Also allowed to be the string 'Any'. |
```

Comment:
I thought we agreed that empty string meant select all, not the string `Any` (which is a valid CloudEvents type)?

Reply:
I agree `Any` is strange. @Harwayne